cmd/catalog/main.go

```diff
@@ -37,7 +37,7 @@ const (
 // config flags defined globally so that they appear on the test binary as well
 var (
 	kubeConfigPath = flag.String(
-		"kubeconfig", "", "absolute path to the kubeconfig file")
+		"kubeconfig", os.Getenv("KUBECONFIG"), "absolute path to the kubeconfig file")
```

Comment:
When providing the value through the CLI flag, that value still takes precedence over the local $KUBECONFIG environment variable, which seems like the correct behavior. Might be worth calling out that behavior in this flags description?